yarn.lock

```diff
@@ -10242,11 +10242,11 @@ utils-merge@1.0.1:
   resolved "https://registry.yarnpkg.com/utils-merge/-/utils-merge-1.0.1.tgz#9f95710f50a267947b2ccc124741c1028427e713"
   integrity sha1-n5VxD1CiZ5R7LMwSR0HBAoQn5xM=
 
-uuid@^3.0.1, uuid@^3.2.1, uuid@^3.3.2:
+uuid@^3.0.1, uuid@^3.3.2:
   version "3.3.2"
   resolved "https://registry.yarnpkg.com/uuid/-/uuid-3.3.2.tgz#1b4af4955eb3077c501c23872fc6513811587131"
```

Comment:
If this is required by a prod dependency we'll want to remove it

Reply:
it's from nyc, snyk, & codecov (all of which are dev-dependencies). should i take any action?

Reply:
No, that's fine.

Reply:
thanks!